pages/seller/js/seller.js

```diff
@@ -73,34 +92,30 @@ let renderProducts = () =>{
     let detail = document.createElement('div');
     detail.className = 'detail';  
     productInfo.appendChild(detail);
-    // console.log(productInfo)
 
-    let names = document.createElement('span');
+    let names = document.createElement('p');
     names.className = 'name';
-    names.textContent = product.name;
+    names.textContent = product.name + ' ' + product.type;
     detail.appendChild(names);
-
-    let type = document.createElement('span');
-    type.className = 'type';
-    type.textContent = product.type;
-    detail.appendChild(type);
-
-    let description = document.createElement('span');
+    //for detailing more information
+    let description = document.createElement('p');
     description.className = 'description';
     description.textContent = product.description;
     detail.appendChild(description);
 
-    let price = document.createElement('span');
+    let price = document.createElement('p');
     price.className = 'price';
-    price.textContent = product.price + ' ' + product.currency;
+    price.textContent = 'Price: ' + product.price + ' ' + product.currency;
```

Comment:
Use modern syntax for concatenation:
names.textContent = 'Price: ' + product.price + ' ' + product.currency;
names.textContent = \`Price : ${product. price}  ${product. currency}\`

Reply:
yes, teacher.
Thank you for your feedback!🙏